virtual-organizations/OSG.yaml

```diff
@@ -181,6 +181,17 @@ DataFederations:
         AllowedCaches:
           - ANY
 
+      - Path: /nrdstor/protected
+        Authorizations:
+          - SciTokens:
+              Issuer: https://t.nationalresearchplatform.org
+              Base Path: /nrdstor/protected
+              Map Subject: True
+        AllowedOrigins:
+          - SDSC_NRP_OSDF_S3_ORIGIN
```

Comment:
Is this the right origin? Perhaps you want NEBRASKA_NRP_HCC_OSDF_ORIGIN ?

Reply:
fixed